PL2/PL2-gui/src/test/java/nl/tudelft/pl2016gr2/gui/view/tree/heatmap/NodeDensityHeatmapTest.java

```diff
@@ -0,0 +1,81 @@
+package nl.tudelft.pl2016gr2.gui.view.tree.heatmap;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import javafx.scene.layout.Pane;
+import javafx.scene.paint.Color;
+import nl.tudelft.pl2016gr2.gui.model.IPhylogeneticTreeNode;
+import nl.tudelft.pl2016gr2.gui.view.tree.Area;
+import nl.tudelft.pl2016gr2.gui.view.tree.ViewNode;
+import nl.tudelft.pl2016gr2.test.utility.AccessPrivate;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.util.ArrayList;
+
+/**
+ * This class tests the {@link ViewNode} class.
```

Comment:
Nice catch. I should probably add this line to my test template so it gets automatically generated.